main/integration-tests/README.md

```diff
@@ -15,7 +15,7 @@
 
 - Create/Locate the test config file in your local repository (placed in `main/integration-tests/config/settings/<STAGE>.yml`). This will be useful when you're triggering the tests locally.
 
-- Enter the Test Administrator username, password path (AWS Parameter Store), Test Project ID, and the rest of the values as directed in the `main/integration-tests/config/settings/example.yml` file. Your config file will be ignored by git when you make any changes.
+- Enter the Test Administrator username, password path (AWS Parameter Store), Test Project ID (Make sure the AWS account the Project ID is assoaicated with has a Budget created through SWB), and the rest of the values as directed in the `main/integration-tests/config/settings/example.yml` file. Your config file will be ignored by git when you make any changes.
```

Comment:
typo 'associated'

Reply:
Started another PR to update README(https://github.com/awslabs/service-workbench-on-aws/pull/382), reverted this one to avoid conflicts.